### PR TITLE
Added Vungle lines and removed dupes of Mobilefuse

### PR DESCRIPTION
**Pull request by adamtimehop**

### CODE REVIEW

File: app-ads.txt [hunk 1]
```diff
@@ -83,6 +80,7 @@ growintech.co, 101696 , RESELLER
 advertising.com,28695, DIRECT, e1a5b5b6e3255540
 mobilefuse.com, 2770, DIRECT, 71e88b065d69c021
 appnexus.com, 2764, RESELLER
+rubiconproject.com,19744, RESELLER, 0bfd66d529a55807
 smartadserver.com, 4037, DIRECT
 rubiconproject.com, 16114, RESELLER, 0bfd66d529a55807
```
Comment (dimamix) [inline app-ads.txt:85]: Line 85: duplicated from line 10

**Comment by dimamix** [comment]
Line 93: "Onetag- onetag.com" is not a valid domain

File: app-ads.txt [hunk 2]
```diff
@@ -83,6 +80,7 @@ growintech.co, 101696 , RESELLER
 advertising.com,28695, DIRECT, e1a5b5b6e3255540
```
Comment (dimamix) [inline app-ads.txt:80]: Line 79: Stray comma

File: app-ads.txt [hunk 3]
```diff
@@ -108,3 +106,44 @@ rubiconproject.com, 17598, RESELLER, 0bfd66d529a55807
...
+tritondigital.com,106423,RESELLER,19b4454d0b87b58b
+targetspot.com,297,RESELLER,feb28ed826dcf532
+bigo.sg,143,RESELLER
+blueseasx.com,203592,RESELLER
+freewheel.tv,1157777,RESELLER
+adtiming.com,a-126,RESELLER,bf66753b8f380142
+freewheel.tv,1446674,RESELLER
+freewheel.tv,1157729,RESELLER
+admixer.co.kr,1094,RESELLER
+App-stock.com,343264,RESELLER
```
Comment (dimamix) [inline app-ads.txt:121]: Domain value contains uppercase characters, consider using lowercase for consistency.

**Comment by dimamix** [comment]
Also ran this through a validator which noted that `Certification authority ID for appnexus.com is: f5ab79cb980f11d1`. Not sure what to do with that warning, was hoping you'd know

File: app-ads.txt (hunk 1 above)
Comment (adamtimehop) [reply app-ads.txt:85]: This one is odd. TappX and Mobilefuse should not have the same reseller lines for rubicon.

File: app-ads.txt [hunk 4]
```diff
@@ -80,9 +76,10 @@ opera.com,pub7275292332480,RESELLER,55a0c5fd61378de3
 e-planning.net,9522026ef023606d,RESELLER,c1ba615865ed87b2
 gitberry.com, 42knb, RESELLER
 growintech.co, 101696 , RESELLER
```
Comment (dimamix) [inline app-ads.txt:78]: @adamtimehop Missed this one - stray comma

**Review by dimamix — APPROVED** [review]

**Pull request merged**